### PR TITLE
Migrate from TSDX to Vite.js

### LINKED ISSUE #106: Replace TSDX

### PR DESCRIPTION
**Pull request by protyze**
- Remove TSDX
- Use vite.js Lib-Mode for bundling/building
- Use vitest instead of jest
- Contains basic toolchain from github.com/netzwerg/react-you-do-you

Closes #106

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,11 +176,10 @@ git push --tags
...
 
-- [TSDX](https://github.com/palmerhq/tsdx) for making TS library development easier
 - [GitHub Corners](https://github.com/tholman/github-corners) for the callout on the example site
 
 ## License
 
 Licensed under [MIT License](LICENSE).
 
-&copy; Rahel Lüthy 2021
+&copy; Rahel Lüthy 2022
```
Comment (netzwerg) [inline README.md:185]: & Jan Azzati :)

**Review by netzwerg — COMMENTED** [review]
Looking good afaict: I was able to run tests & storybook, build, publish (via yalc), and QA kicked-in during pre-commit ✅

Your "basic demo" is not working 100% (weird font, no tooltips), but I guess we should migrate to proper, isolated stories anyway.

**Review by netzwerg — COMMENTED** [review]
Why is this not needed anymore?

**Comment by protyze** [comment]
It is needed, but within the .github folder (not ./workflows/main.yml -> that was a copy/past mistake). I kept the original workflow and removed the one from react-you-do-you.

**Comment by protyze** [comment]
> Your "basic demo" is not working 100% (weird font, no tooltips), but I guess we should migrate to proper, isolated stories anyway.

That's just because there is no caption fontFamily defined in the default-theme and there is no tooltip labels in the test-data. The example is really just to demonstrate that "it's working" and we can create more advanced demos within stories.

**Pull request merged**